docs/source/noise.rst

```diff
@@ -884,7 +885,7 @@ where
    p(x') & = \sum_{x\in\mathcal{O}} p(x'|x) p(x)  \\
    & = \tr{\sum_{x\in \mathcal{O}} p(x'|x) \Pi_x \rho \Pi_x} \\
    & = \tr{\rho \sum_{x\in \mathcal{O}} p(x'|x)\Pi_x} \\
-   & = \tr{\rho E_x'}.
+   & = \tr{\rho E_{x'}}.
```

Comment:
I am far too "mathematically challenged" to be sure whether this is an actual typo, or just me being confused. I suspect I'm just confused, but figured I'd include it and let someone sort me out in code review.

Reply:
Link to the corresponding docs section:

http://docs.rigetti.com/en/stable/noise.html#classical-readout-bit-flip-error

Reply:
Looks good to me

Reply:
To be clear, my confusion here is whether the difference between `E_x'` and `E_{x'}` is just a typo, or whether they really are two distinct but valid things being talked about in this set of equations.

I am particularly suspicious because before my changes, both `E_x'` and `E_{x'}` appear exactly twice, and it seems weird that someone would make the same typo in 2 out of 4 places. Additionally, since both appear twice, I'm not sure which of the two is the typo, even assuming they are all meant to refer to the same object (if that makes sense).

Things that make me suspect it might be a typo:

1. Equations don't make sense to me otherwise. This is a very weak argument, though :)
2. If someone really wanted `E_x'`, I would expect them to spell it `E'_x`

Reply:
I mean your changes look good. I'm convinced because `E_{x'}` is defined in the previous line, whereas `E_{x}'` is defined nowhere.